onboard/nav/stateMachine.hpp

```diff
@@ -88,6 +88,10 @@ class StateMachine
 
     void addRepeaterDropPoint();
 
+    void updateRepeaterDropWaypoint();
+
+    void lowRadioSignalStrengthTime();
```

Comment:
Make this function name start with a verb. Since that's going to make it longer, let's also try to shorten the name a little.